octodns/provider/cloudflare.py

```diff
@@ -512,7 +588,11 @@ def _gen_key(self, data):
         # BUT... there are exceptions. MX, CAA, LOC and SRV don't have a simple
         # content as things are currently implemented so we need to handle
         # those explicitly and create unique/hashable strings for them.
-        _type = data['type']
+        # AND... for URLFWD/Redirects additional adventures are created.
+        if 'targets' in data:
+            _type = 'URLFWD'
+        else:
+            _type = data['type']
```

Comment:
If `"type"` doesn't exist in `data` for URLFWD this can probably be simplified to

```suggestion
        _type = data.get('type', 'URLFWD')
```